can/interfaces/pcan/pcan.py

```diff
@@ -83,6 +88,13 @@ def __init__(self, channel, *args, **kwargs):
         if result != PCAN_ERROR_OK:
             raise PcanError(self._get_formatted_error(result))
 
+        if win32event:
+            self._recv_event = win32event.CreateEvent(None, 0, 0, None)
+            result = self.m_objPCANBasic.SetValue(
```

Comment:
Just confirming that the pcan api dictates that this must be a win32event? We can't pass in a more standard Python type such as `threading.Event`?